testsuite/tests/typing-layouts-products/basics_unboxed_records.ml

```diff
@@ -161,7 +161,9 @@ Line 4, characters 2-7:
 4 |   left'
       ^^^^^
 Error: This value is "local"
-       because it is from the allocation (at Line 2, characters 10-25) containing a value
+       because it is the field "left" of the record at Line 3, characters 6-25
+       which is "local"
+       because it is allocated at Line 2, characters 10-25 with content
```

Comment:
```suggestion
       because it is allocated at Line 2, characters 10-25 pointing to a value
```

? Not sure. But `with content` felt a little jarring to me.